### PR TITLE
Update password-policy-recommendations.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the minimum to 8 characters as to encourage users to think about a unique password, MIcrosoft recommends keeping a reasonable 8-character minimum length requirement.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7941

File: microsoft-365/admin/misc/password-policy-recommendations.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ Good password practices fall into a few broad categories:
 
 The primary goal of a more secure password system is password diversity. You want your password policy to contain lots of different and hard to guess passwords. Here are a few recommendations for keeping your organization as secure as possible.
   
-- Maintain a 14-character minimum length requirement
+- Maintain a 8-character minimum length requirement
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/misc/password-policy-recommendations.md]:
```suggestion
- Maintain an eight-character minimum length requirement
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this Pr. Thanks

**Pull request merged**